inclusive-design/idrc
# feat: add 404 (resolves #10)

## Description

Adds 404 page.

## Steps to test

1. Visit deploy preview and attempt to load a page which doesn't exist.

**Expected behavior:** 404 page is displayed.

## Additional information

- https://www.11ty.dev/docs/quicktips/not-found/
- https://docs.netlify.com/routing/redirects/redirect-options/#custom-404-page-handling

## Related issues

- Resolves #10.
